rasa_nlu/train.py

```diff
@@ -62,7 +62,7 @@ def init():     # pragma: no cover
     return config
 
 
-def do_train(config, component_builder=None):
+def do_train(config, component_builder=None, in_worker=False):
```

Comment:
I don't like functions with varying signatures (in this case the return type).

Let's keep this function as it was and add a wrapper function here that only returns the persisted path (e.g. `do_train_in_worker`).